api/v1alpha1/nstemplatetier_types.go

```diff
@@ -45,6 +50,12 @@ type NSTemplateTierClusterResources struct {
 	TemplateRef string `json:"templateRef"`
 }
 
+// NSTemplateTierSpaceRoles the space roles definition in an NSTemplateTier resource
+type NSTemplateTierSpaceRoles struct {
```

Comment:
btw, the name should have been singular, and not plural 
```suggestion
type NSTemplateTierSpaceRole struct {
```

Reply:
changed in cdea401